README.md

```diff
@@ -1,7 +1,15 @@
 # rancher/hardened-build-base
 
+This repository holds the Dockerfiles and builds scripts for [rancher/hardened-build-base](https://hub.docker.com/r/rancher/hardened-build-base) Docker images. The x86_64 image contains a FIPS compatible Go compiler ([Go+BoringCrypto](https://github.com/golang/go/tree/dev.boringcrypto/misc/boring)), used for [compiling rke2 components](https://docs.rke2.io/security/fips_support/#fips-support-in-cluster-components).
+
+Supported architectures are x86_64/amd64, arm64, and s390x.
```

Comment:
No arm64 support yet.

Reply:
That is for trivy and for future-proofing for when we're capable of doing arm64.